### DISCUSSION TITLE
hugging face model not really working

### DISCUSSION THREAD

**Discussion by mistersprinklez** [TheLastBen/fast-stable-diffusion#1301] [General]
Hi!

I had some very good luck with a model when testing on the gradio server, but when I upload it to hugging face it's only generating really bad results. 

Any idea what's happening with hugging face upload?

**Comment by TheLastBen** [comment]
the diffusers inference lacks some features as opposed to A1111 gradio webui

**Reply by JOHNFFFEE** [reply]
what do you think the best way, to overcome the issue?
we currently loading in our backend hf model.
I installed gradio on our backend, but the images remains the same

**Reply by TheLastBen** [reply]
You need the A1111 repo

**Reply by JOHNFFFEE** [reply]
do you mean that I need to use webui and customize it , or do you have a repo I can clone, to load the weights and return an image to our frontend

**Reply by TheLastBen** [reply]
A1111 has an API feature, check their wiki
